src/main/java/com/zebrunner/carina/demo/websocket/EndpointUtils.java

```diff
@@ -0,0 +1,81 @@
+package com.zebrunner.carina.demo.websocket;
+
+import com.zebrunner.carina.utils.Configuration;
+import com.zebrunner.carina.utils.report.ReportContext;
+import com.zebrunner.carina.webdriver.listener.DriverListener;
+import org.openqa.selenium.TimeoutException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.remote.RemoteWebDriver;
+import org.openqa.selenium.support.ui.FluentWait;
+
+import java.net.Authenticator;
+import java.util.List;
+import java.util.Optional;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public final class EndpointUtils {
+
+    private EndpointUtils() {
+        //hide
+    }
+
+    public static String getWebSocketSelenoidUrl(WebDriver driver, String endpoint) {
```

Comment:
let's remove selenoid from the method name

Reply:
done